client/app/index.html

```diff
@@ -177,7 +177,7 @@
                   <b><translate>Price</translate></b>
                 </div>
 
-                <span class="md-secondary">{{ul.currency.symbol}}{{ul.connectedPeer.market.price[ul.currency.name]}}</span>
+                <span class="md-secondary">{{ul.formatCurrencyVal(ul.connectedPeer.market.price[ul.currency.name])}}</span>
```

Comment:
These look like good use cases for a custom angular filter

Reply:
yeah! I agree @spresnal